src/Mvc/Mvc.Analyzers/src/ViewFeaturesAnalyzerContext.cs

```diff
@@ -32,11 +32,6 @@ public bool IsHtmlHelperExtensionMethod(IMethodSymbol method)
                 return false;
             }
 
-            if (method.ReceiverType != HtmlHelperType)
-            {
-                return false;
-            }
-
```

Comment:
This appears to be a behavior difference between operations and symbols. When I get the symbol through an `IOperation` the receiver is the class (static extension method), the existing code expect the receiver to be the `this IHtmlHelper helper` parameter.

/cc @mavasani in case he has any idea

Reply:
Either way I don't think checking this is critical for our success.